coalib/core/Bear.py

```diff
@@ -130,6 +132,21 @@ class Bear:
     CAN_FIX = set()
     ASCIINEMA_URL = ''
     BEAR_DEPS = set()
+    """
+        Attributes for a new bear.
+
+        :param section:
+            The section object where bear settings are contained.
+        :param file_dict:
+            The file-dictionary containing a mapping of filenames to the
+            according file contents.
+        :raises RuntimeError:
+            Raised when bear requirements are not fulfilled.
+    """
+    section = attr.ib(hash=False)
+    file_dict = attr.ib(hash=False)
+    _dependency_results = attr.ib(init=False, default=defaultdict(list),
+                                  hash=False)
```

Comment:
@Makman2 just two cents here: `attrs` generates comparisons/string representations "quickly" and only gives slight overhead at class definition but not at runtime. These might be the "selling points" you are trying to hear.

As for this particular use case, `attrs` probably does not differ much from just writing the `__init__` manually. You could argue, that you do not have to implement `__repr__` and `__str__` and can even compare bears with `__le__` and the like.